# Reports the solenoid and control state of the solenoid_ctrl package
bool sol_ctrl        # indicates whether key_sub is in control mode
bool sol_bear        # indicates the status of the air bearing solenoid valve
int16[] sol_thrust   # indicates the status of the thruster air bearings

